etc/ci/check_no_unwrap.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#
+# Make sure listed files do not contain "unwrap"
+set -e
+
+cd `git rev-parse --show-toplevel` # cd into repo root so make sure paths works in any case
```

Comment:
Please use `$(git rev-parse --show-toplevel)` instead of backticks for process substitution.